test/service/test_pagination.py

```diff
@@ -111,16 +111,17 @@ def test_search_after_page2(self):
         :return:
         """
         # Fetch and check first page.
-        content = requests.get("{}?sort=entryId&order=desc".format(self.get_base_url())).content
+        content = requests.get(self.get_base_url() + '?sort=entryId&order=desc').content
         json_response = json.loads(content)
         self.assert_page1_correct(json_response)
 
         # Fetch the second page using search_after
         search_after = json_response['pagination']['search_after']
         search_after_uid = json_response['pagination']['search_after_uid']
-        url = "{}?sort=entryId&order=desc&search_after={}&search_after_uid={}".format(self.get_base_url(),
-                                                                                      search_after, search_after_uid)
-        content = requests.get(url).content
+        content = requests.get(f'{self.get_base_url()}'
+                               f'?sort=entryId&order=desc'
+                               f'&search_after={search_after}'
+                               f'&search_after_uid={search_after_uid}').content
```

Comment:
A bit cleaner, but not as uniform with all f-strings. Your thoughts? I could agree if you think all f-strings is the better approach here.
```suggestion
        content = requests.get(self.get_base_url() +
                               f'?sort=entryId&order=desc'
                               f'&search_after={search_after}'
                               f'&search_after_uid={search_after_uid}').content
```

Reply:
I considered this and thought it was distractingly irregular. But now I'm having second thoughts. I think I'll commit it separately and leave it up to Hannes.